src/PolkitDialog.vala

```diff
@@ -186,11 +185,7 @@ namespace Ag.Widgets {
             idents_combo.active = 0;
 
             if (length < 2) {
```

Comment:
Would it be a ton of work to do && admin user != this user?

Reply:
I don't think much, no.